src/src/services/hasura/hasura.service.ts

```diff
@@ -381,4 +381,48 @@ export class HasuraService {
 			return { [tableName]: result };
 		}
 	};
+
+	public async getDataFromSQL(sql: string) {
+		try {
+			let url = this.configService.get<string>('HASURA_SQL_BASE_URL');
+			let admin_secret = this.configService.get<string>(
+				'HASURA_ADMIN_SECRET',
+			);
+			const data = {
+				type: 'run_sql',
+				args: {
+					source: 'educategirlsdev',
```

Comment:
what is source here?

Reply:
educategirlsdev will this work on uat, prod?

Reply:
Source is database name.
Added database name in env. Now, getting it from that.